osv2/search.go

```diff
@@ -61,14 +64,15 @@ func V2AggregateConverter(agg opensearchtools.Aggregation) (opensearchtools.Aggr
 	return agg, nil
 }
 
-// NewSearchRequest instantiates a SearchRequest with a Size of -1.
-// Any negative value for SearchRequest.Size will be ignored and not included in the source.
+// NewSearchRequest instantiates a SearchRequest with a From and Size of -1.
+// Any negative value for SearchRequest.From or SearchRequest.Size will be ignored and not included in the source.
```

Comment:
```suggestion
// Any negative value for [SearchRequest.From] or [SearchRequest.Size] will be ignored and not included in the source.
```
Wrapping the field references in [] will make them clickable hyperlinks in the generated docs.